crates/cargo-wdk/README.md

```diff
@@ -1,117 +1,115 @@
 
-You can test the installation by running:
-```pwsh
-cargo wdk --version
-```
+Options:
+      --kmdf  Create a KMDF driver crate
+      --umdf  Create a UMDF driver crate
+      --wdm   Create a WDM driver crate
+  -h, --help  Print help
 
-For help, run:
-```pwsh
-cargo wdk --help
+Verbosity:
+  -v, --verbose...  Increase logging verbosity
+  -q, --quiet...    Decrease logging verbosity
 ```
 
-## Installing WDK
+`new` takes the type of driver project you want to create (`kmdf`, `umdf` or `wdm`) and its destination path (`PATH`) as inputs along with flags specifying log verbosity.
 
-`cargo-wdk` builds the drivers using the WDK. Please ensure that the WDK is installed on the development system.
-The recommended way to do this is to [enter an eWDK developer prompt](https://learn.microsoft.com/en-us/windows-hardware/drivers/develop/using-the-enterprise-wdk#getting-started).
+The last component of `PATH` is used as the name of the crate.
 
-## Usage Examples
+#### Examples
 
-1. `new` command to create a new Rust driver project: 
-    ```pwsh
-    cargo wdk new [OPTIONS] <DRIVER_PROJECT_PATH>
+- To create a new KMDF project called `my_driver` under the current folder run:
+
+    ```pwsh  
+    cargo wdk new my_driver --kmdf  
     ```
-    
-    Example Usage:
 
-    * Create a UMDF project called `sample_driver` under the folder `my_projects`  
+- To create a new UMDF project called `my_driver` under the folder `my_projects` run:  
+
     ```pwsh  
-    cargo wdk new my_projects\sample_driver --umdf  
+    cargo wdk new my_projects\my_driver --umdf  
     ```  
 
-    * Create a KMDF project called `sample_driver` under the current folder  
-    ```pwsh  
-    cargo wdk new sample_driver --kmdf  
-    ``` 
+### `build` Command
 
-    Use `--help` for more information on arguments and options
+```pwsh
+Usage: cargo wdk build [OPTIONS]
+
+Options:
+      --profile <PROFILE>          Build artifacts with the specified profile
+      --target-arch <TARGET_ARCH>  Build for the target architecture
+      --verify-signature           Verify the signature
+      --sample                     Build sample class driver project
+  -h, --help                       Print help
+
+Verbosity:
+  -v, --verbose...  Increase logging verbosity
+  -q, --quiet...    Decrease logging verbosity
+```
 
-2. `build` command to build and package driver projects:
-    ```pwsh
-    cargo wdk build [OPTIONS]
-    ```
-    
-    Example Usage: 
-    * Navigate to the project/workspace root and run
+`build` takes a number of inputs specifying build profile (`dev` or `release`), target architecture (`amd64` or `arm64`), a flag enabling signature verification and a flag indicating a sample driver along with verbosity flags.
 
-        ```pwsh 
-        cargo wdk build 
-        ```
+When the command completes the packaged driver artifacts are emitted at the path `target\<profile>\<project-name>-package`.
 
-    * With `--target-arch`
+#### Workspace support
```

Comment:
The old content has a mention of emulated/nested workspaces. Looks like it is missed here.